spec/models/instrument_price_policy_calculations_spec.rb

```diff
@@ -299,7 +299,7 @@
       reservation.actual_end_at = reservation.reserve_end_at - 10.minutes
       new_costs = policy.calculate_cost_and_subsidy reservation
       expect(new_costs[:cost]).to eq(61.to_d)
-      expect(new_costs[:subsidy]).to eq(8.6083335.to_d)
+      expect(new_costs[:subsidy].round(2)).to eq(8.6083335.to_d.round(2))
```

Comment:
Why is this changing?

Reply:
@giladshanan I'm not certain regarding the need for this. But the expectation does fail now due to rounding differences, whereas it didn't before. Even though the value of `new_costs[:subsidy]` is `0.86083335e1` both before and after the upgrade.

```
> expect(new_costs[:subsidy]).to eq(8.6083335.to_d)
RSpec::Expectations::ExpectationNotMetError: 
expected: 0.8608333500000001e1
     got: 0.86083335e1

(compared using ==)